fea-rs/src/compile/variations.rs

```diff
@@ -14,9 +19,20 @@ pub trait VariationInfo {
     fn axis_info(&self, axis_tag: Tag) -> Option<AxisInfo>;
 
     /// Return the normalized value for a user coordinate for the given axis.
+    ///
+    /// NOTE: This is only used for computing the normalized values for ConditionSets.
     fn normalize_coordinate(&self, axis_tag: Tag, value: Fixed) -> F2Dot14;
+
+    /// return the default value, and then the regions and their deltas.
+    fn resolve_variable_metric(
+        &self,
+        locations: &HashMap<UserLocation, i16>,
```

Comment:
IMHO we should land https://github.com/googlefonts/fontc/pull/347 and had fea-rs use fontdrasil types for coordinates and conversions between them once it supports unit suffixes. Right now half the compilers code will be type conversions. I want to use those suffixes in https://github.com/googlefonts/fontc/pull/345.

Reply:
I went back and forth, and ended up just keeping things as minimal as possible. Once we support different coordinate spaces this API will definitely need to be rethought.

Reply:
Also we presumably to be able to pass in any location type, so perhaps the input is a map from something like an AnyLocation enum so you can provide in whatever units fea specified?

Reply:
yep that's definitely a reasonable option, or I could imagine a separate normalization step.

Reply:
If you want to just provide normalized values that works too. Ideally as the same NormalizedLocation type fontc uses :)

Reply:
I'm trying to feed this so I normalized is what I actually want

```rust
    pub fn deltas<P, V>(
        &self,
        point_seqs: &HashMap<NormalizedLocation, Vec<P>>,
    ) -> Result<Vec<(VariationRegion, Vec<V>)>, DeltaError>
    where
        P: Copy + Default + Sub<P, Output = V>,
        V: Copy + Mul<f64, Output = V> + Sub<V, Output = V>,
```